includes/Images/ImageSettings.php

```diff
@@ -83,6 +84,11 @@ private function register_settings() {
 									),
 								),
 							),
+							'bulk_optimization' => array(
```

Comment:
Can we move this option above lazy loading to have it be next to the other optimization ones?